python/pypto/ir/__init__.py

```
@@ -111,8 +115,23 @@
     "op_conversion",
     "register_op_conversion",
     "make_roundtrip_instrument",
+    "directions",
+    "make_call",
 ]  # fmt: skip
 
+# Re-export the lowercase ArgDirection aliases at the top level so callers can
+# spell ``ir.input``, ``ir.output``, etc. without importing the submodule.
+# These intentionally shadow the builtin ``input`` name; users only ever access
+# them via ``ir.input``, which keeps the global builtin untouched.
+input = _directions.input
```

Ruff (0.15.10)

[error] Variable `input` is shadowing a Python builtin

(A001)